pkg/debug/scripts/collect-info.sh

```diff
@@ -45,6 +58,30 @@ while getopts "vh" o; do
     esac
 done
 
+# We are not on EVE? Switch to read-logs mode
+if [ ! -d "/persist" ]; then
+    FIND=".log"
+
+    if [ ! -z "$READ_LOGS_DEV" ]; then
+        FIND="dev.log"
+    elif [ ! -z "$READ_LOGS_APP" ]; then
+        FIND="app.$READ_LOGS_APP.log"
+    fi
+
+    # Find compressed logs (device or application or both)
+    find . -name \*$FIND\*gz | \
+        # Extract timestamp and make it a first column
+        perl -ne 'if (/\.(\d+).gz$/){ printf "$1 $_";  }' | \
+        # Sort
+        sort -n | awk '{print $2}' | \
+        # Decompress and output
+        xargs --no-run-if-empty zcat | \
+        # Concat timestamp and make it human readable
+        jq  '.timestamp.seconds |= strftime("%B %d %Y %I:%M:%S") | .timestamp = "\(.timestamp.seconds).\(.timestamp.nanos)"'
+
+    exit
+fi
+
```

Comment:
@rouming , some information that I still miss a lot from collect-info.sh is a list of /dev directory to see which file devices are present. I think we could take this opportunity to add this information (output of `ls -l /dev`), would you mind to make this change as well in this PR?

Reply:
Yep, will add.

Reply:
The SC2156 is explitly ignored in the code, but yetus still complains. I don't have any desire to fight with the tool, so leave it as is.